app/_includes/md/kic/prerequisites.md

```diff
@@ -36,9 +36,9 @@ Kubernetes exposes the proxy through a Kubernetes service. Run the following com
 1. Populate `$PROXY_IP` for future commands:
 
     ```bash
-    export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong kong-gateway-proxy)
-    
-    echo "Proxy IP: $PROXY_IP"
+    HOST=$(kubectl get svc --namespace kong kong-gateway-proxy -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    PORT=$(kubectl get svc --namespace kong kong-gateway-proxy -o jsonpath='{.spec.ports[0].port}')
```

Comment:
Why do we add the port? Don't loadbalancers always listen on port 80 by default?

Reply:
I just added the commands that I saw after installing using Helm.